src/kafka/inflight_activation_batcher.rs

```diff
@@ -40,7 +48,15 @@ impl Reducer for InflightActivationBatcher {
     type Output = Vec<InflightActivation>;
 
     async fn reduce(&mut self, t: Self::Input) -> Result<(), anyhow::Error> {
-        self.buffer.push(t);
+        let runtime_config = self.runtime_config_manager.read().await;
+        let task_name = &t.activation.taskname;
+        if !runtime_config.drop_task_killswitch.contains(task_name) {
```

Comment:
I realize I should have thought of this on your previous PR, but it would be nice if this was a HashSet() instead of a Vec for these contains checks. In practice it's probably not a big deal, but just in case.